Adding Registry and Portal to networking graphic.

stuclem opened the pull request:
Per title. Also tarted up the graphic a bit, to try to distinguish better better between what is VIC, what is vSphere, and what is Docker.

@mlh78750 can you please take a look? Thanks!

stuclem commented:
Fixes https://github.com/vmware/vic-product/issues/16.

mlh78750 commented:
Sorry for the delay on the review. While the position of Harbor and Admiral on the management network is a valid setup, it's not the only way this would work. The only requirement for the registry is that the endpoint VM has a usable route to the registry over any network. Practically speaking this will be the management or the public network. For Admiral, it just needs a usable route to the endpoint VM's client network's interface. @hmahmood to check my answers.

hmahmood commented:
@mlh78750 is correct. You can move the Admiral box in the image to connect to the client network. The registry can be attached to the client, public, or management networks, although the latter two is the recommended approach. Not sure how we would represent that in the picture though.

stuclem commented:
Thanks @mlh78750 and @hmahmood. I have moved the VIC Appliance to the left of the image, and connected the Management Portal to the client network, leaving Registry connected to the management network. I updated the text of the topic that includes this graphic to state which networks Registry/MPortal can/must connect to. Changes are in https://github.com/vmware/vic-product/pull/189/commits/6d9278e844cb825cfef244b13e6bbbfbd21db4fe. Is this OK now?

hmahmood commented on docs/user_doc/vic_vsphere_admin/networks.md:
```diff
@@ -16,33 +16,33 @@ This topic provides an overview of the different network types that virtual cont
 ## Management Network {#management}
 
-The network for communication between the VCH and vCenter Server and ESXi hosts. The VCH uses this network to provide the `attach` function of the Docker API.
+The network for communication between the VCH, vCenter Server, and ESXi hosts. The VCH uses this network to provide the `attach` function of the Docker API. vSphere Integrated Containers Registry must be able to access VCHs either over the management network or over the public network.
```
Isn't this the over way around? i.e. the VCH needs to access the VIC registry over any of the networks the VCH is connected to (management and public preferred).

stuclem commented:
Thanks @hmahmood. Duly updated.

stuclem commented:
Thanks @hmahmood !
